src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -88,6 +88,7 @@ public override TypeSyntax VisitArrayType(IArrayTypeSymbol symbol)
 
                         break;
                     }
+#endif
```

Comment:
this is one area i did need to change code.  That's beacuse `.NullableAnnotation` doesn't seem to be avialable in this layer yet.  i assume we would have to wait to be able to reference a more recent compiler version.

Reply:
if we move forward with this PR we should have a tracking issue to remove these.